tsan.suppressions

```diff
@@ -1 +1 @@
-race:event_debug_node_too_late
+race:event_debug_mode_too_late
```

Comment:
Excellent! I plan to referring abseil::ANNOTATE_BENIGN_RACE but it requires patch on each event
What does `event_debug_mode_too_late` mean?

Reply:
It is a monotonic variable which indicates that it is now too late to enable event_debug_mode.

It isn't important for regular behavior and since it is monotonic the race is benign.

Reply:
unfortunately I can't test locally because of my version of clang and I can't get the supression to work on circleci.   Arg.

Reply:
You're missing another `p` in `suppressions=`.

Reply:
done